### PR TITLE
CI build script experiments

### PR DESCRIPTION
**Pull request by tpoterba**
here we go

please have your SIGKILL at the ready

### CODE REVIEW

**Comment by danking** [comment]
🔫

File: hail/hail-ci-build.sh [hunk 1]
```diff
@@ -44,29 +60,51 @@ trap on_exit EXIT
...
+export GRADLE_OPTS="-Xmx2048m"
+export GRADLE_USER_HOME="/gradle-cache"
+
+echo "Compiling..."
+./gradlew shadowJar archiveZip > ${COMPILE_LOG}
+
+test_project() {
+    ./gradlew test > ${SCALA_TEST_LOG}
+    ./gradlew testPython > ${PYTHON_TEST_LOG}
+    ./gradlew doctest > ${DOCTEST_LOG}
```
Comment (danking) [inline hail/hail-ci-build.sh]: we should be able to run all of these in parallel too, right?

Comment (tpoterba) [reply hail/hail-ci-build.sh]: how many system resources do these things have?

Comment (tpoterba) [reply hail/hail-ci-build.sh]: and I don't think it will matter much -- this will still finish before VEP I think

Comment (danking) [reply hail/hail-ci-build.sh]: We're using n1-standard-8's and we reserve about half of the resources for each PR job. We could use bigger machines if it speeds up builds though.

**Comment by tpoterba** [comment]
removed the parallelism; will follow up with that after.

artifacts page looks like this:

image


There are some more changes I want to make, but I'll break it up.

**Comment by tpoterba** [comment]
Also I have banished the serifs defiling the artifacts page as well.

**Comment by tpoterba** [comment]
Yeah, I poked at that a bit without a solution.

**Pull request merged**